tools/src/wyvern/tools/tests/effectSeparation/file1.wyv

```diff
@@ -0,0 +1,6 @@
+module file1 : {} effectSeparation.File
```

Comment:
I believe tests that use `file1`, `filedef3`, and `filedef4` should fail providing an error message about not all of their methods having effect annotations on them.